include/SongEditor.h

```diff
@@ -141,11 +143,13 @@ private slots:
 
 	PositionLine * m_positionLine;
 
-	ComboBoxModel* m_zoomingModel;
+	ComboBoxModel* m_zoomingXModel;
+	ComboBoxModel* m_zoomingYModel;
 	ComboBoxModel* m_snappingModel;
 	bool m_proportionalSnap;
 
-	static const QVector<float> m_zoomLevels;
+	static const QVector<float> m_zoomXLevels;
+	static const QVector<float> m_zoomYLevels;
```

Comment:
```suggestion
	static constexpr std::array m_zoomXLevels = { 0.125f, 0.25f, 0.5f, 1.0f, 2.0f, 4.0f, 8.0f, 16.0f };
	static constexpr std::array m_zoomYLevels = { 1.0f, 1.25f, 1.5f, 1.75f, 2.0f, 2.5f, 3.0f, 4.0f, 8.0f, 16.0f };
```
Can use constexpr arrays instead.